Doc/library/os.path.rst

```diff
@@ -306,17 +306,18 @@ the :mod:`glob` module.)
 
 .. function:: join(path, *paths)
 
-   Join one or more path components intelligently.  The return value is the
+   Join one or more path segments intelligently.  The return value is the
    concatenation of *path* and any members of *\*paths* with exactly one
```

Comment:
```suggestion
   concatenation of *path* and all members of *\*paths* with exactly one
```
"all" sounds clearer here. Maybe "any" was meant to emphasize that `paths` can be empty?

Reply:
Thanks, also seems like there should be a comma so the "with" can't be interpreted as applying to "all members of paths"
```suggestion
   concatenation of *path* and all members of *\*paths*, with exactly one
```